pysat/utils/io.py

```diff
@@ -51,9 +51,10 @@ def pysat_meta_to_xarray_attr(xr_data, pysat_meta, epoch_name):
 
                 # Cycle through all the pysat MetaData labels and transfer
                 for meta_key in pysat_meta[data_key].keys():
-                    # Assign attributes
-                    xr_data[xarr_vars[i]].attrs[meta_key] = pysat_meta[
-                        data_key][meta_key]
+                    # Assign attributes with values that are not None
```

Comment:
Did this change arise from a bug? Couldn't there be a situation where a user wants the value to be None and in the file?

Reply:
This was changed as part of a bugfix in in https://github.com/pysat/pysat/pull/1095